tests/installer/default.nix

```diff
@@ -40,7 +40,7 @@ pkgs.nixosTest {
 
     print(machine.succeed("tty"))
 
-    machine.succeed("disko-install --disk ${disk} /dev/vdb --flake ${self}#${testConfig}")
+    machine.succeed("disko-install --write-efi-boot-entries --disk ${disk} /dev/vdb --flake ${self}#${testConfig}")
```

Comment:
@remimimimimi that's a very subtle difference. Let me fix this in nixpkgs actually.
The issue is that disko-install needs to override the NixOS configuration to disable writing efi vars as the default because it would be not fit as a USB flasher if it would make the system it's running on unbootable.

Reply:
That in turn triggered a rebuild of systemd-boot-builder.